GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -138,7 +138,7 @@ public bool TryKillRunningProcess()
 
                     if (process != null)
                     {
-                        process.Kill();
+                        GVFSPlatform.Instance.KillProcessTree(process.Id);

```

Comment:
The idea was to `kill` only background processes launched by VFS4Git, right? I don't see a check.

Reply:
@alameenshah the process is only called from GitMaintenanceStep.  This a buyer beware type of process.